internal/updater/updater.go

```diff
@@ -84,13 +83,6 @@ func (u *AvailableUpdate) prepareInstall(installTargetPath string, args []string
 		return "", nil, errs.Wrap(err, "Downloaded update does not have installer")
 	}
 
-	if installTargetPath == "" {
-		installTargetPath, err = installation.InstallPath()
-		if err != nil {
-			return "", nil, errs.Wrap(err, "Could not detect install path")
-		}
-	}
```

Comment:
I could see why this would be problematic for `--set-channel`, but won't this cause issues for the default flow?

Reply:
Yep, you're right. I've put it back.